android/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -829,6 +845,54 @@ private void finishPlayerInitialization() {
         }
     }
 
+    private static ExoMediaDrm createFrameworkMediaDrm(UUID uuid) {
+        try {
+            return VirtuosoFrameworkMediaDrm.newInstance(uuid);
+        } catch (UnsupportedDrmException e) {
+            e.printStackTrace();
+            return null;
+        }
+    }
+
+    private DrmSessionManager buildDrmSessionManager(String assetId) {
+        if (assetId == null) {
+            return null;
+        }
+        Virtuoso virtuoso = new Virtuoso(getContext());
+
+        IAsset asset = null;
+
+        List<IIdentifier> list = virtuoso.getAssetManager().getByAssetId(assetId);
+        if (list != null && list.size() > 0) {
+            asset = (IAsset) list.get(0);
```

Comment:
If the asset can't be found, that seems like an error we should through with a clear message, not something we should try and continue with, using a null drm manager.